hal/armv7m/stm32/l4/Makefile

```diff
@@ -12,12 +12,9 @@ LDFLAGS := $(filter-out -Tdata% , $(LDFLAGS))
 CFLAGS := $(filter-out -mfloat-abi% , $(CFLAGS))
 CFLAGS += -mfloat-abi=soft
 
-PLO_COMMANDS := alias app bankswitch call console copy dump echo go help kernelimg map mem mpu \
+PLO_COMMANDS ?= alias app bankswitch call console copy dump echo go help kernelimg map mem mpu \
   phfs reboot script wait
 
-#include devices/pipe-rtt/Makefile
-include devices/uart-stm32l4x6/Makefile
-include devices/flash-stm32/Makefile
-include devices/ram-storage/Makefile
+PLO_ALLDEVICES := uart-stm32l4x6 flash-stm32 ram-storage # pipe-rtt
```

Comment:
do you know why `pipe-rtt` is commented out?

Is it not supported (than remove it) or it wasn't desirable to add it to compilation in reference project by default (than maybe use the new mechanism by specifying `PLO_DEVICES` in project file explicitly?)

Reply:
It is fully supported on stm32l4 (I use it on nucleo with integrated stlink instead of uart console and usb for phfs), it would be nice to have it enabled by default, so that CI can track changes.
I assume that now with these changes, an internal project using stm32l4 will be able to "undefine" RTT in plo if it does not need it.

Reply:
So could it be enabled by default in remaining architectures? 
imxrt105x
stm32l4x6

Or it should be keep disabled on imxrt105x?

Reply:
I'd enable it if it fits under the kernel currently

Reply:
Keep disabled on i.MX RT105x (also `RTT_ADDR` for RT105x is not defined in `peripherals.h`) due to small amount of space in DTCM (single ocram/flexram as opposed to 106x and 1176).